### PR TITLE
Fix error schema field updateat

### PR DESCRIPTION
**Pull request by alavieille**
[OO-BUGFIX] Fix error type of property `updateAt` in content type schema

### CODE REVIEW

File: ElasticaAdminBundle/Resources/translations/messages.fr.yml [hunk 1]
```diff
@@ -16,3 +16,5 @@ open_orchestra_elastica_admin:
     form:
         elastica_list:
             search_limit: Limite de résultat
+        elastica_search:
+            node: Page qui affiche les résultats
```
Comment (ngilain) [inline ElasticaAdminBundle/Resources/translations/messages.fr.yml]: `Nœud d'affichage des résultats` is a better translation I think

**Comment by ngilain** [comment]
I think a OO-BUGFIX tag is required here

**Comment by alavieille** [comment]
@itkg-ngilain pr updated

**Comment by ngilain** [comment]
:+1:

**Pull request merged**